simulations/vip-193/vip-193/simulation.ts

```diff
@@ -0,0 +1,113 @@
+import { TransactionResponse } from "@ethersproject/providers";
+import { impersonateAccount } from "@nomicfoundation/hardhat-network-helpers";
+import { expect } from "chai";
+import { Contract } from "ethers";
+import { ethers } from "hardhat";
+
+import { expectEvents, setMaxStalePeriodInChainlinkOracle } from "../../../src/utils";
+import { forking, testVip } from "../../../src/vip-framework";
+import { vip193 } from "../../../vips/vip-193/vip-193";
+import PRIME_ABI from "./abis/Prime.json";
+import PRIME_LIQUIDITY_PROVIDER_ABI from "./abis/PrimeLiquidityProvider.json";
+import SETTER_FACET_ABI from "./abis/SetterFacet.json";
+
+const PRIME_LIQUIDITY_PROVIDER = "0x103Af40c4C30A564A2158D7Db6c57a0802b9217A";
+const PRIME = "0x78d8dD5b0003723826E1FDb2031e9466000469Fe";
+const STAKED_USER = "0xb70d7b0cebb570e9007039b9cfd6034822f81758";
+const CHAINLINK_ORACLE = "0x1B2103441A0A108daD8848D8F5d790e4D402921F";
+const NORMAL_TIMELOCK = "0x939bD8d64c0A9583A7Dcea9933f7b21697ab6396";
+
+interface vTokenConfig {
+  name: string;
+  assetAddress: string;
+  feed: string;
+}
+
+const vTokens: vTokenConfig[] = [
+  {
+    name: "vUSDC",
+    assetAddress: "0x8AC76a51cc950d9822D68b83fE1Ad97B32Cd580d",
+    feed: "0x51597f405303c4377e36123cbc172b13269ea163",
+  },
+  {
+    name: "vUSDT",
+    assetAddress: "0x55d398326f99059fF775485246999027B3197955",
+    feed: "0xb97ad0e74fa7d920791e90258a6e2085088b4320",
+  },
+  {
+    name: "vETH",
+    assetAddress: "0x2170Ed0880ac9A755fd29B2688956BD959F933F8",
+    feed: "0x9ef1b8c0e4f7dc8bf5719ea496883dc6401d5b2e",
+  },
+  {
+    name: "vBTC",
+    assetAddress: "0x7130d2A12B9BCbFAe4f2634d864A1Ee1Ce3Ead9c",
+    feed: "0x264990fbd0a4796a3e3d8e37c4d5f87a3aca5ebf",
+  },
+  {
+    name: "vXVS",
+    assetAddress: "0xcF6BB5389c92Bdda8a3747Ddb454cB7a64626C63",
+    feed: "0xbf63f430a79d4036a5900c19818aff1fa710f206",
+  },
+];
+
+forking(33118003, () => {
+  describe("Pre-VIP behavior", () => {
+    let prime: Contract;
+    let primeLiquidityProvider: Contract;
+
+    before(async () => {
+      impersonateAccount(STAKED_USER);
+      const signer = await ethers.getSigner(STAKED_USER);
+      prime = await ethers.getContractAt(PRIME_ABI, PRIME, signer);
+      primeLiquidityProvider = await ethers.getContractAt(PRIME_LIQUIDITY_PROVIDER_ABI, PRIME_LIQUIDITY_PROVIDER);
+    });
+
+    it("prime markets", async () => {
+      expect((await prime.getAllMarkets()).length).to.equal(0);
+    });
+
+    it("prime address", async () => {
+      expect(await primeLiquidityProvider.prime()).to.equal("0x0000000000000000000000000000000000000000");
+    });
+
+    it("claim prime token", async () => {
+      await expect(prime.claim()).to.be.reverted;
+    });
+  });
+
+  testVip("VIP-193 Prime Program", vip193(), {
+    callbackAfterExecution: async (txResponse: TransactionResponse) => {
+      await expectEvents(txResponse, [SETTER_FACET_ABI], ["NewPrimeToken"], [1]);
```

Comment:
I would not check this in this VIP, because at the end that command won't be part of this VIP